src/main/java/org/fenixedu/cms/domain/Sanitization.java

```diff
@@ -57,8 +60,20 @@ public static LocalizedString sanitize(LocalizedString origin) {
         return result;
     }
 
+    public static LocalizedString strictSanitize(LocalizedString origin) {
+        LocalizedString result = new LocalizedString();
+        for (Locale l : origin.getLocales()) {
+            result = result.with(l, sanitize(origin.getContent(l)));
+        }
+        return result;
+    }
+
     public static String sanitize(String original) {
         return sanitizer.apply(original);
     }
 
+    public static String strictSanitize(String original) {
+        return strictSanitizer.apply(original);
```

Comment:
return STRICT_SANITIZER.sanitize(original);

This would have the same result with the benefit of not needing to have an extra static variable, thus reducing the applications footprint and the resulting byte code would be more efficient because you will have one less indirection to invoke the underlying sanitize() method.

The same criticism is applicable to the use of CMS_SANITIZER.sanitize(origin).